src/utils/fetchClient.ts

```diff
@@ -13,6 +13,7 @@ type RequestMethod = 'GET' | 'POST' | 'PATCH' | 'DELETE';
 function request<T>(
   url: string,
   method: RequestMethod = 'GET',
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   data: any = null, // we can send any data to the server
```

Comment:
Avoid using any type. In TypeScript, the any type is used as a last resort when we cannot declare a specific type. It's better to use more specific type for 'data' parameter or at least an unknown type.